### ISSUE TITLE
Can't find optical flow data

### ISSUE THREAD

**Issue by cghero-fi** [lhoangan/eden-generation#1]
I am really interested on your EDEN datasets, but I can't find the optical flow of the datasets on the website. Thanks!

**Comment by lhoangan** [comment]
Hi, 
Thank you for your interest. Optical flow data takes up an enormous space and contain mostly ego-motion (as we only have static scenes) so it turns out not very interesting for sharing. If it suits your problem, I could try to share a sample scene in raw data format (*.exr) so you'll have to extract them into flow format. Please let me know.

**Comment by cghero-fi** [comment]
Can you share some sample data to me, I think it may be helpful to my problems, Thanks.
This is my email: raymond880215.cs10@nycu.edu.tw

**Comment by ChristophReich1996** [comment]
Hi @lhoangan, I would be very interested in using your dataset with optical flow labels. Is there any way to still share the full optical flow labels?

**Comment by lhoangan** [comment]
Hi,

Thank you for your interest. I am not sure if the optical flow would be of any use because these are static scenes and the flows are all from the camera's ego-motion. Besides, I've lost connection to the data repository due to changes of affiliation. Could you please try the data [here](https://isis-data.science.uva.nl/hale/EDEN/OpticalFlow.zip) which is the only that I have. Please do let me know if it's what you are looking for.

Regards

**Comment by ChristophReich1996** [comment]
Hi, Thanks a lot for your very fast reply! Unfortunately, the link does not work. Can you maybe suggest someone to reach out to who still has access to the dataset?
We want to use your dataset for optical flow pre-training. While the dataset only contains ego-motion still, learning fine optical flow patterns of trees, etc., could be facilitated using your dataset. Especially since datasets like TartanAir contain significant errors in their optical flow labels, accurate ground truth flow labels would help us :)

**Comment by lhoangan** [comment]
Hi @ChristophReich1996 , I'm sorry that that file isn't downloadable, it may be due to a permission problem on the server. I've made a request to the admin since a few days and, sorry, that's the best that I can do for now. I will get back to you when it's done, but it may take some time. Most people are going on vacation around this time of the year.

**Comment by ChristophReich1996** [comment]
@lhoangan Thanks for your update; looking forward!

**Comment by lhoangan** [comment]
Hi @ChristophReich1996  @cghero-fi, the OpticalFlow data at https://isis-data.science.uva.nl/hale/EDEN/OpticalFlow.zip are downloadable now but only limited to the 12 scenes corresponding to those from the 3DRMS challenge (thus around 100 locations x 5 cameras x 12 scenes = 6K images) . That's all that I can provide for now. A word of caution: could you please confirm by replying if the data are, at least, correct (corresponding well to the given RGB images) and impossible, if you find them usable for your work? Feel free to update me with other information if you want. I'm quite interested in knowing if the data are useful. Thank you.

**Comment by ChristophReich1996** [comment]
Hi @lhoangan, amazing! Thanks for making the optical flow labels available. I just downloaded the data and will keep you updated about the correctness of the labels. Thanks again :)

**Comment by lhoangan** [comment]
Hi @ChristophReich1996 I'm wondering if you had a chance to test the data? I'm still interested in knowing if they're usable.
